client/src/pages/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,6 @@
+.login-container {
```

Comment:
Change to `LoginPage.modules.css` and make any necessary changes to the files that use this CSS file
Refer to the comment made on the first CSS file that I commented on

Reply:
Fixed (Nestor): changed to .module